compiler/rustc_error_codes/src/error_codes/E0744.md

```diff
@@ -12,12 +12,14 @@ const _: i32 = {
 };
 ```
 
-At the moment, `if` and `match`, as well as the looping constructs `for`,
-`while`, and `loop`, are forbidden inside a `const`, `static`, or `const fn`.
+At the moment, `for` loops, `.await`, and the `Try` operator (`?`) are
+forbidden inside a `const`, `static`, or `const fn`.
 
-This will be allowed at some point in the future, but the implementation is not
-yet complete. See the tracking issue for [conditionals] or [loops] in a const
-context for the current status.
+This may be allowed at some point in the future, but the implementation is not
+yet complete. See the tracking issues for [`impl const Trait for Ty`], [`&mut
+T` in `const fn`], [`async` in `const fn`], and [`?` in `const fn`].
```

Comment:
Why are you mentioning `&mut` here?

Reply:
On the [const eval tree](https://rust-lang.github.io/const-eval/), `&mut T` is marked as a prerequisite for having `for` loops/iterators

Reply:
That is true, but I do not think that should affect the docs here. The tracking issue for `for` loops should point this out, though.

Reply:
Oh, looks like there is no tracking issue specifically for for loops? I see.

So what about something like "See the tracking issues for `async` and `?` in `const fn`, and (to support `for` loops in `const fn`) the tracking issues for `impl const Trait for Ty` and `&mut T` in `const fn`".

Reply:
Yeah, sounds good. I was wondering how to word that better.